modules/ROOT/pages/questions/question-18.adoc

```diff
@@ -0,0 +1,12 @@
+18. https://ephocks.atlassian.net/browse/TINY-4603[TINY-4603] notes
```

Comment:
@HAFRMO @ltrouton

Reply:
> Should this be documented entirely separately or is it safe to just note the re-naming also applies to `autolink`?

I am inclined to say it should be the latter. Setting the `link_default_target` option will affect both `autolink` and `link` plugins i.e. the default link target cannot be controlled separately for each of the plugins